Update log destination docs (#8242)
fleetdm/fleet: docs/Deploying/Configuration.md

##### osquery_status_log_plugin

> Which log output plugin should be used for osquery status logs received from clients. Check out the reference documentation for log destinations [here in the Fleet documentation](../Using-Fleet/Log-destinations.md).

Reviewer: ```suggestion
This is the log output plugin that should be used for osquery status logs received from clients. Check out the [reference documentation for log destinations](../Using-Fleet/Log-destinations.md).
```

##### osquery_result_log_plugin

> Which log output plugin should be used for osquery result logs received from clients. Check out the reference documentation for log destinations [here in the Fleet documentation](../Using-Fleet/Log-destinations.md).

Reviewer: ```suggestion
This is the log output plugin that should be used for osquery result logs received from clients. Check out the [reference documentation for log destinations](../Using-Fleet/Log-destinations.md).
```
